### DISCUSSION TITLE
Jaeger UI Local Setup Issue on Windows

### DISCUSSION THREAD

**Discussion by kirtanchandak** [jaegertracing/jaeger#4557] [General]
### Hey, I am trying to setup [jaeger-ui](https://github.com/jaegertracing/jaeger-ui) on my local system but facing this error while installing can someone please help me out?

![jag-ui](https://github.com/jaegertracing/jaeger/assets/92196705/40183425-afdd-4669-a023-65ea755753b3)

### I have http://localhost:16686/ already setup on my system - 

![Screenshot 2023-07-02 145202](https://github.com/jaegertracing/jaeger/assets/92196705/144e1ae5-fb42-4dcf-a952-8f7d6d651c31)

**Comment by albertteoh** [comment]
Just trying to understand, why do you want to setup jaeger-ui when you have http://localhost:16686/ setup?

**Comment by kirtanchandak** [comment]
http://localhost:16686/ is running from the docker container and I need to run the react code inside the  jaeger-ui for the hot reload whenever I make any changes!

**Comment by kirtanchandak** [comment]
I am trying to solve this issue - https://github.com/jaegertracing/jaeger-ui/issues/15 can you help me out with where can I see the preview of these copy icons, I am new to contributing to this organization!

**Comment by albertteoh** [comment]
> I am trying to solve this issue - https://github.com/jaegertracing/jaeger-ui/pull/15 can you help me out with where can I see the preview of these copy icons, I am new to contributing to this organization!

I guess you meant this issue: https://github.com/jaegertracing/jaeger-ui/issues/1506?

> http://localhost:16686/ is running from the docker container and I need to run the react code inside the jaeger-ui for the hot reload whenever I make any changes!

Thanks, makes sense. I think windows nvm doesn't recognise the `.nvmrc` file, but you're doing the right thing with `nvm use 16` I think.

Have you tried the suggestions from the `yarn install` output? i.e. remove the `package-lock.json` file, and running in verbose mode (i.e. `yarn install --verbose`)?

Worth mentioning that when I run `nvm use`, I have the following version installed (not sure it would matter though):

```
$ nvm current
v16.15.0
```

**Comment by kirtanchandak** [comment]
### Facing this error now!
![verbose](https://github.com/jaegertracing/jaeger/assets/92196705/2599abce-bf8f-4587-b962-49ea7e1eabde)

**Reply by yurishkuro** [reply]
in both cases your command just fails without giving any error message ("failed with exit code 1"), so I don't have any idea how to troubleshoot it. `yarn install` works for me on Mac, and it works in our CI on Ubuntu.

**Comment by anikdhabal** [comment]
This is because '$npm_execpath' is not recognized as an internal or external command . Try to replace it with %npm_execpath% if you are in windows. Follow this for more info https://stackoverflow.com/questions/53883405/npm-execpath-isnt-recognized-as-an-internal-or-external-command-program-or

**Reply by yurishkuro** [reply]
would it help if we replaced this line with `yarn build`? 

```
    "prepublishOnly": "$npm_execpath build",
```
https://github.com/jaegertracing/jaeger-ui/blob/c179d57cff329726ebbd8d20ddc6b12a100a46ca/packages/plexus/package.json#L68

It's the only place that uses `$npm_execpath`

**Reply by anikdhabal** [reply]
Then problem occurs 'NODE_ENV' is not recognized.
